### ISSUE TITLE
Storemanager.load is not a function (?)

### ISSUE THREAD

**Issue by WinterVein** [TyruswooStudio/Tyruswoo_AltimitMovement-MZ#3]
When trying to use the plugin the game crashes on start saying storemanager.load is not a function. what does this mean?

the error message more specifically
`rmmz_managers.js:2036 TypeError: StorageManager.load is not a function
    at Function.CollisionMesh.getMesh (Tyruswoo_AltimitMovement0.7.js:3224)
    at Game_Map.collisionMesh (Tyruswoo_AltimitMovement0.7.js:2902)
    at VectorDebugLayer.update (altimitmovementDEBUG.js:128)
    at Spriteset_Map.Sprite.update (rmmz_core.js:1987)
    at Spriteset_Map.Spriteset_Base.update (rmmz_sprites.js:3158)
    at Spriteset_Map.update (rmmz_sprites.js:3379)
    at Scene_Map.createSpriteset (rmmz_scenes.js:1009)
    at Scene_Map.createDisplayObjects (rmmz_scenes.js:1000)
    at Scene_Map.onMapLoaded (Zoom.js:298)
    at Scene_Map.isReady (rmmz_scenes.js:777)`

**Comment by McKathlin** [comment]
Hi @WinterVein, thanks for bringing this to our attention!

It turns out that the crash you encountered only happens when caching is turned on, and that if I fix this particular crash, other caching problems show up that also cause crashes.

For now, the workaround I recommend is to turn caching off:
1. Open the Plugin Manager and go into Tyruswoo_AltimitMovement.
2. Find the plugin parameter "Use cached collision?" at the bottom of the plugin list, under the "Play-testing" heading.
3. Set "Use cached collision" to No (false).

Please let me know if you run into any further issues after turning off caching.

**Comment by McKathlin** [comment]
Hi @WinterVein, we hope game development is going well for you.

We reviewed our plugin's feature of caching to files and determined that it could not be made usable except by a complete overhaul of mesh calculation, and that even if it were fixed, it was unlikely to confer significant benefit. Accordingly, our newly released [Version 0.8.0](https://github.com/TyruswooStudio/Tyruswoo_AltimitMovement-MZ/releases/tag/v0.8.0) removes caching from this plugin.

Thanks again for keeping us informed of issues!
